examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method allows to calculate different quantities
+inside blocks/bins whose dimensions are defined via the ``spacing`` parameter.
+The following example shows how to calculate averages of given values inside
+each block as well as how to report the number of points inside each bin.
+"""
+
+import pygmt
+
+# Load sample data
+data = pygmt.datasets.load_japan_quakes()
+# Select only needed columns
+data = data[["longitude", "latitude", "depth_km"]]
+
+# Set the region for the plot
+region = [130, 152.5, 32.5, 52.5]
+# Define spacing in x and y direction (150 by 150 minute blocks)
+spacing = "150m"
+
+fig = pygmt.Figure()
+
+# Calculate mean depth in km from all events within 150x150 minute
+# bins using blockmean
+df = pygmt.blockmean(data, region=region, spacing=spacing)
+# convert to grid
+grd = pygmt.xyz2grd(df, region=region, spacing=spacing)
+
+fig.grdimage(
+    grd,
```

Comment:
```suggestion
    grid=grd,
```
Explicitly passing an argument to a parameter